pkg/apis/batch/validation/validation_test.go

```diff
@@ -585,6 +605,27 @@ func TestValidateCronJob(t *testing.T) {
 				},
 			},
 		},
+		"spec.jobTemplate.spec.template.spec.restartPolicy: Required value": {
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "mycronjob",
+				Namespace: metav1.NamespaceDefault,
+				UID:       types.UID("1a2b3c"),
+			},
+			Spec: batch.CronJobSpec{
+				Schedule:          "* * * * ?",
+				ConcurrencyPolicy: batch.AllowConcurrent,
+				JobTemplate: batch.JobTemplateSpec{
+					Spec: batch.JobSpec{
+						Template: api.PodTemplateSpec{
+							Spec: api.PodSpec{
```

Comment:
set `RestartPolicy: api.RestartPolicyAlways,` here to make sure we don't accidentally start allowing `Always`

Reply:
Thanks @liggitt !
Done.